test/WebSites/LocalizationWebSite/Models/User.cs

```diff
@@ -7,7 +7,7 @@ namespace LocalizationWebSite.Models
 {
     public class User
     {
-        [MinLength(4, ErrorMessage = "NameError")]
+        [MinLength(6, ErrorMessage = "NameError")]
```

Comment:
I changed the number because it used to hard coded into the localization and I wanted to make sure I wasn't getting the correct answer for the wrong reasons.